admin/src/components/VendorTable.tsx

```diff
@@ -43,18 +42,18 @@ const VendorTable = () => {
         <tbody>
           {allVendors.map((vendor) => {
             return (
-              <tr key={vendor.id}>
-                <td>{vendor.name}</td>
+              <tr key={vendor["id"]}>
```

Comment:
Ez elég csúnya hack, így pont a típusbiztosság beszik el :) Lehet a useState-nek is type annotationt adni: https://www.carlrippon.com/typed-usestate-with-typescript/
Illetve célszerű lenne akár egy külön file-ba bevezetni egy Vendor interface-t, amit lehetne használni később több helyen. (https://www.typescriptlang.org/docs/handbook/modules.html)

Reply:
Bocsi a sok hackelős megoldásért, csak nagyon új még a Typescript és minden errorra 4-5 különböző megoldási javaslatot találok, és még nem tudom eldönteni, hogy melyik a normális megoldás közülük.

Megcsináltam amúgy a kiszervezést, nem tudom, hogy ilyen megoldásra gondoltál-e. Én már nagyon kevés dologban vagyok biztos :D